types/jest/index.d.ts

```diff
@@ -424,33 +424,79 @@ declare namespace jest {
         each: Each;
     }
 
+    type PrintLabel = (string: string) => string;
+
+    type MatcherHintColor = (arg: string) => string;
+
+    interface MatcherHintOptions {
+        comment?: string;
+        expectedColor?: MatcherHintColor;
+        isDirectExpectCall?: boolean;
+        isNot?: boolean;
+        promise?: string;
+        receivedColor?: MatcherHintColor;
+        secondArgument?: string;
+        secondArgumentColor?: MatcherHintColor;
+    }
+
     interface MatcherUtils {
         readonly isNot: boolean;
         readonly dontThrow: () => void;
         readonly promise: string;
+        readonly assertionCalls: number;
+        readonly expectedAssertionsNumber: number | null;
+        readonly isExpectingAssertions: boolean;
+        readonly suppressedErrors: any[];
+        readonly expand: boolean;
+        readonly testPath: string;
+        readonly currentTestName: string;
         utils: {
-            readonly EXPECTED_COLOR: (text: string) => string;
-            readonly RECEIVED_COLOR: (text: string) => string;
-            diff: typeof import('jest-diff');
-            ensureActualIsNumber(actual: any, matcherName?: string): void;
-            ensureExpectedIsNumber(actual: any, matcherName?: string): void;
-            ensureNoExpected(actual: any, matcherName?: string): void;
-            ensureNumbers(actual: any, expected: any, matcherName?: string): void;
-            /**
-             * get the type of a value with handling of edge cases like `typeof []` and `typeof null`
-             */
-            getType(value: any): string;
+            readonly EXPECTED_COLOR: import("chalk").Chalk & {
+              supportsColor: import("chalk").ColorSupport;
+            };
+            readonly RECEIVED_COLOR: import("chalk").Chalk & {
+              supportsColor: import("chalk").ColorSupport;
+            };
+            readonly INVERTED_COLOR: import("chalk").Chalk & {
+              supportsColor: import("chalk").ColorSupport;
+            };
+            readonly BOLD_WEIGHT: import("chalk").Chalk & {
+              supportsColor: import("chalk").ColorSupport;
+            };
+            readonly DIM_COLOR: import("chalk").Chalk & {
+              supportsColor: import("chalk").ColorSupport;
+            };
+            readonly SUGGEST_TO_CONTAIN_EQUAL: string;
```

Comment:
The chalk import should go the top of the file and only be imported once

Reply:
Awesome, cheers.

Reply:
@Favna That doesn't seem to be working.

Is there a disadvantage to doing how it was previously aside from being a bit repetitive, and if not, is this something we could fix/improve straight after this PR instead of blocking here?

I don't mind making the changes, but I'd like to get these published for usage ideally asap & am aware of how busy people can be :)

Reply:
Confirmed it didn't work. Didn't *expect* it to break that badly.